maxim/logger/fireworks/utils.py

```
@@ -0,0 +1,347 @@
+            "frequency_penalty",
+            "perf_metrics_in_response",
+            "presence_penalty",
+            "repetition_penalty",
+            "top_k",
+            "min_p",
+            "response_format",
+            "reasoning_effort",
+            "stream",
+            "n",
+            "stop",
+            "context_length_exceeded_behavior"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)